elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -128,7 +133,7 @@ def attach_cve_flaws_cli(runtime, advisory_id, noop, default_advisory_type):
 
     if noop:
         print('DRY-RUN: The following changes would have been applied to the advisory:')
-        print(advisory)
+        print(advisory.dump())
```

Comment:
i observed `advisory.dump()` seems to have a bug:
```
  File "/workspaces/elliott/elliottlib/cli/attach_cve_flaws_cli.py", line 136, in attach_cve_flaws_cli
    print(advisory.dump())
  File "/home/lmeyer/.local/lib/python3.9/site-packages/errata_tool/erratum.py", line 996, in dump
    print("Manager Email: " + self.manager_email)
TypeError: can only concatenate str (not "NoneType") to str
```
`manager_email` should be set there. that's nothing wrong with this code, just something we might want to consider reverting if it causes problems.

Reply:
Ah yes, I remember hitting from problem now. I think I removed offending line from erratum.py but that means it doesn't print the manager_email. If we don't use dump we don't get the updated description section of the erratum.